<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Drugs::Internal::DosageCreatorDialog</class>
 <widget class="QDialog" name="Drugs::Internal::DosageCreatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="testOnlyButton">
       <property name="text">
        <string>For test only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prescribeButton">
       <property name="toolTip">
        <string>The dosage will be used for your actual prescription,&lt;br /&gt; but will not be saved in your dosage database.</string>
       </property>
       <property name="text">
        <string>Only prescribe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" rowspan="2" colspan="2">
    <widget class="DosageViewer" name="dosageViewer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="interactionIconLabel">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="drugNameLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DrugName</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="Views::ListView" name="availableDosagesListView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="toolTip">
        <string>The dosage will be saved in your dosage databse, &lt;br /&gt;no cahnges will be done to your actual prescription.</string>
       </property>
       <property name="text">
        <string>Save changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveAndPrescribeButton">
       <property name="toolTip">
        <string>The dosage will be used for your actual prescription,&lt;br /&gt; and it will be saved in your dosage database.</string>
       </property>
       <property name="text">
        <string>Save and prescribe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Cancel this window, all changes will be lost.</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DosageViewer</class>
   <extends>QWidget</extends>
   <header location="global">dosagedialog/mfDosageViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Views::ListView</class>
   <extends>QListView</extends>
   <header location="global">listviewplugin/listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>availableDosagesListView</tabstop>
  <tabstop>prescribeButton</tabstop>
  <tabstop>helpButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Drugs::Internal::DosageCreatorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
